app/src/services/mongo/MongoDatabaseConnectionService.ts

```diff
@@ -0,0 +1,31 @@
+import ServiceInterface from '../../interfaces/ServiceInterface';
+import {getLogger} from '../../logger';
+import {injectable} from 'inversify';
+import MongoDBConnection from './MongoDBConnection';
+
+@injectable()
+export default class MongoDatabaseConnectionService implements ServiceInterface {
+    private static readonly logger = getLogger('MongoDatabaseConnectionService');
+
+    private readonly dbClient: MongoDBConnection;
+
+    constructor(dbClient: MongoDBConnection) {
+        this.dbClient = dbClient;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/require-await
+    public async boot(): Promise<void> {
+        MongoDatabaseConnectionService.logger.info('Booting Mongo Database Connection Service...');
+    }
+
+    // eslint-disable-next-line @typescript-eslint/require-await
+    public async start(): Promise<void> {
+        MongoDatabaseConnectionService.logger.info('Starting Mongo Database Connection Service...');
+        this.dbClient.getConnection();
```

Comment:
I think this needs to be in the boot method as the database should be ready to go once the application starts.

Reply:
Yeah, I think so too. Or we need an additional function to update different values provided by the database

Reply:
Moved to boot method, but it doesn't appear to remain within boot. Suggestions welcome

Reply:
Figured it out